demo/index.html

```diff
@@ -110,7 +110,10 @@ <h1 class="c-main__title">React Components</h1>
               <a class="u-fg-inherit" href="storybook/?path=/story/components-chrome-readme--page">Chrome</a>
             </div>
             <div class="u-mb-sm">
-              <a class="u-fg-inherit" href="storybook/?path=/story/components-datepickers-readme--page">Datepickers</a>
+              <a class="u-fg-inherit" href="storybook/?path=/story/components-colorpicker-readme--page">Colorpicker</a>
```

Comment:
This makes the first color larger by 2 component entries. Should we rebalance and move `Dropdowns` to the top of the next column?

Reply:
Updated